File: MODUL-5/Bahasa Python/PRAK501-2310817120014-ALIYARAFFANAURAAYU.py
def MaxBilangan(w, x, y, z):
    if w > x and w > y and w > z:
        return w
    elif x > w and x > y and x > z:
        return x
    elif y > w and y > x and y > z:
        return y
    else:
        return z

a, b, c, d = input().split()
hasil = MaxBilangan(a, b, c, d)
print(hasil)

File: MODUL-5/Bahasa C/PRAK501-2310817120014-ALIYARAFFANAURAAYU.c
#include <stdio.h>

//Buatlah Function Disini
int MaxBilangan (int w, int x, int y, int z){
    if (w > x && w > y && w > z){
        return w ;}
    else if (x > w && x > y && x > z){
        return x ;}
    else if (y > w && y > x && y > z){
        return y ;}
    else {
        return z ;}
        }
        
int main() {
int a, b, c, d; 
scanf("%d %d %d %d", &a, &b, &c, &d);
int hasil = MaxBilangan(a, b, c, d);
printf("%d", hasil); 
return 0;
}
